source/Generic/PlayState/PlayState.cs

```diff
@@ -120,6 +119,14 @@ public override void OnApplicationStarted(OnApplicationStartedEventArgs args)
                     .Show();
                 ProcessStarter.StartUrl(@"https://github.com/darklinkpower/PlayniteExtensionsCollection/wiki/PlayState#window-notification-style-configuration");
             }
+
+            var suspendPlaytimeOnlyFeature = PlayniteApi.Database.Features.FirstOrDefault(a => a.Name.Equals("[PlayState] Suspend Playtime only", StringComparison.OrdinalIgnoreCase));
+            if (suspendPlaytimeOnlyFeature != null)
```

Comment:
Ignoring if the name should be changed or not, this check should only be done once by saving a value in settings, otherwise it will be done always on startup